large-message-core/src/main/java/com/bakdata/kafka/AbstractLargeMessageConfig.java

```diff
@@ -73,51 +75,57 @@
  * <ul>
  *     <li> Connection string
  * </ul>
+ * <p></p>
+ * Google Cloud Storage specific
+ * <ul>
+ *     <li> Project ID
+ * </ul>
  */
 @Slf4j
```

Comment:
I think there are some formatting problems, because our d9p checkstyle formatter has a different continuation indentation.

Reply:
No, I was using the outdated [bakdata codestyle](https://github.com/bakdata/bakdata-code-styles/blob/master/intellij/codestyles/bakdata.xml)